test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -44,6 +44,14 @@ public void VerifyImages_2_0(string osVariant)
             VerifyImages(dotNetCoreVersion: "2.0", osVariant: osVariant);
         }
 
+        [Fact]
+        [Trait("Version", "2.0")]
+        [Trait("Architecture", "ARM")]
+        public void VerifyImages_2_0_Arm()
```

Comment:
What prevents this test from either no-oping or not running in a Windows container environment?